src/utils/SanitizeStringValues.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Removes the unnecessary quotation marks in the string
```

Comment:
We do not know, at least I don't, if it is unnecessary or not. It might be added for some purpose in `markdown-transform`.

Reply:
@algomaster99 
Does the description `Strips off the quotation marks from the first and last position` sound right then?

Reply:
You can just omit `unnecessary`.

Reply:
Yes, you are correct. You can change it to what you suggested above.

Reply:
@algomaster99 
Updated the description. Let me know if there are any changes required.